pytorch_lightning/trainer/connectors/model_connector.py

```diff
@@ -38,9 +38,6 @@ def copy_trainer_model_properties(self, model):
             m.testing = self.trainer.testing
             m.precision = self.trainer.precision
 
-    def get_model(self):
-        return self._get_reference_model(self.trainer.model)
-
     def _get_reference_model(self, model):
```

Comment:
this can be removed too? it's used only once in line: 28 up there. can just use `self.trainer.lightning_module`

Reply:
@rohitgr7 Thanks. I addressed your suggestion in e9c4e2c